pkg/kubefedctl/join.go

```diff
@@ -228,13 +242,13 @@ func JoinCluster(hostConfig, clusterConfig *rest.Config, kubefedNamespace,
 	}
 
 	klog.V(2).Infof("Performing preflight checks.")
-	err = performPreflightChecks(clusterClientset, joiningClusterName, hostClusterName, kubefedNamespace, errorOnExisting)
+	err = performPreflightChecks(clusterClientset, joiningClusterName, hostClusterName, joiningNamespace, errorOnExisting)
 	if err != nil {
 		return err
 	}
 
 	klog.V(2).Infof("Creating %s namespace in joining cluster", kubefedNamespace)
```

Comment:
It is trying to create`joiningNamespace` namespace in joining cluster. 
Same comments for all related parts: when we use `joiningNamespace`, we should log `joiningNamespace`.

Reply:
Done.